engines/director/lingo/lingo-code.cpp

```diff
@@ -57,11 +57,12 @@ static struct FuncDescr {
 } funcDescr[] = {
 	{ 0,					"STOP",			"" },
 	{ Lingo::c_xpop,		"c_xpop",		"" },
-	{ Lingo::c_arraypush,	"c_arraypush",		"i" },
+	{ Lingo::c_argspush,	"c_argspush",	"i" },
+	{ Lingo::c_arraypush,	"c_arraypush",	"i" },
 	{ Lingo::c_printtop,	"c_printtop",	"" },
-	{ Lingo::c_constpush,	"c_constpush",	"i" },
+	{ Lingo::c_intpush,	"c_intpush",	"i" },
```

Comment:
What is the purpose of renaming `c_constpush` and `c_fconstpush`?

Reply:
The naming was slightly confusing; Lingo scripts have a constants table and there are opcodes which push objects from that table using indexes, as distinct from pushing an integer/float encoded in the program listing.